pygem/params.py

```diff
@@ -294,6 +299,79 @@ def print_info(self):
 		print '\nposition_vertex_3 ='
 		print self.position_vertex_3
 
+	def build_bounding_box(self, shape, tol=1e-6, triangualte=False, triangulate_tol=1e-1):
+		min_xyz, max_xyz = self._calculate_bb_dimension(shape, tol, triangualte, triangulate_tol)
+		self.origin_box = min_xyz
+		self._set_box_dimensions(min_xyz, max_xyz)
+		self._set_position_of_vertices()
+		self._set_mapping()
+		self._set_transformation_params_to_zero()
+
+	def _set_box_origin(self, xyz):
```

Comment:
Please use the decorator for setters and getters like this. [Here](http://docs.quantifiedcode.com/python-anti-patterns/correctness/implementing_java-style_getters_and_setters.html) you can find a good guide.